pages/sponsor.tsx

```diff
@@ -0,0 +1,171 @@
+                further down. We're open to discuss requests outside our listed
+                offerings.
+              </P>
+              <P>
+                We've published 6 episodes in the last 90 days with a total of
+                11k podcast plays, 2.5k site page views, and 12k YouTube views.
+                In the last 60 days we've had 130k views on TikTok. In the last
+                30 days we've had 102k impressions on twitter with 7k profile
+                visits.
+              </P>
+
+              <H2>Offerings</H2>
+              <H3>1st - ($500)</H3>
+              <Ul>
+                <li>
+                  60 second ad slot at the beginning of each bi-weekly episode
+                  released during the partnership period
+                </li>
+                <li>
+                  Logo, links, and messaging on devtools.fm homepage during the
+                  partnership period
+                </li>
+                <li>
+                  Logo, links, and messaging on sponsored episode pages which
+                  remains even after the partnership period ends
+                </li>
+                <li>
+                  Refreshed social posts on sponsor's episode (if applicable)
+                </li>
+                <li>Sponsored links on episode social promotions</li>
+                <li>
+                  Consideration of collaboration opportunities or special
+                  requests
+                </li>
+                <li>Input on guest lineup</li>
+              </Ul>
+              <H3>2nd - ($300)</H3>
+              <Ul>
+                <li>
+                  30 second ad slot at the midpoint of each bi-weekly episode
+                  released during the partnership period
+                </li>
+                <li>
+                  Logo, links, and messaging on devtools.fm homepage during the
+                  partnership period
+                </li>
+                <li>
+                  Logo, links, and messaging on sponsored episode pages which
+                  remains even after the partnership period ends
+                </li>
+              </Ul>
+              <H2>Terms</H2>
+              <P>
+                We're asking the above prices per episode for an initial
+                partnership duration between 4 to 8 episodes. With longer
+                partnerships we will also lower the price accordingly. Typical
+                industry rates are around $30 per 1k listeners for a 60 second
+                ad slot. We trend above that price point depending on the
+                popularity of the guest but a consistent rate allows us to
+                reliably reinvest in the show to further its growth which in
+                turn should increase the ROI for the partnership. Our guests are
+                also known developers with interest in trying and buying tools,
+                so you address a very specific market.
+              </P>
```

Comment:
I think we could drop the justification piece at the end here. Having sponsors is sort of social proof enough of it being worth sponsoring so I don't think it'll be necessary to further justify our asking rate.